main.cpp

```diff
@@ -71,7 +71,8 @@ int main(int argc, char* argv[]) {
     context->setContextProperty("rotationExecutorBreakdownModel", gui_control->get_rotation_executor_model());
     context->setContextProperty("rotationExecutorListModel", gui_control->get_rotation_executor_list_model());
     context->setContextProperty("simScaleModel", gui_control->get_sim_scale_model());
-    context->setContextProperty("scaleResultModel", gui_control->get_scale_result_model());
```

Comment:
There are still some references to scaleResultModel in StatisticsScaleResultSorting.qml which causes errors. You'll probably need to pass the model it should use for its operations.

Reply:
hmm. Wanna commandeer this commit? Not sure how to parameterize that QML... type? to pass down the right `ScaleResultModel`

Reply:
I'll accept a QML error for now and fix it later as it is fairly minor.